pkg/sfu/buffer/buffer.go

```diff
@@ -28,7 +28,7 @@ type pendingPacket struct {
 type ExtPacket struct {
 	Head      bool
 	Arrival   int64
-	Packet    rtp.Packet
+	Packet    *rtp.Packet
```

Comment:
I do wonder why it was storing the packet itself rather than a pointer.. my guess is the struct is fairly small so it's likely negligible

Reply:
It is not super small. It has the full RTP header
```
// Header represents an RTP packet header                                                                                                       
 16 type Header struct {                                                                                                                            
 17     Version          uint8                                                                                                                      
 18     Padding          bool                                                                                                                       
 19     Extension        bool                                                                                                                       
 20     Marker           bool                                                                                                                       
 21     PayloadType      uint8                                                                                                                      
 22     SequenceNumber   uint16                                                                                                                     
 23     Timestamp        uint32                                                                                                                     
 24     SSRC             uint32                                                                                                                     
 25     CSRC             []uint32                                                                                                                   
 26     ExtensionProfile uint16                                                                                                                     
 27     Extensions       []Extension                                                                                                                
 28 }                                                                                                                                               
 29                                                                                                                                                 
 30 // Packet represents an RTP Packet                                                                                                              
 31 type Packet struct {                                                                                                                            
 32     Header                                                                                                                                      
 33     Payload     []byte                                                                                                                          
 34     PaddingSize byte                                                                                                                            
 35 }
```